### PR TITLE
Need feedback on this one

### PR DESCRIPTION
**Pull request by saraswatpuneet**
This is a spin off PR from PR https://github.com/LibertyDSNP/mrc/pull/109
@enddynayn  Should we rename all "delegate" to "provider" based off your comment on PR above

### CODE REVIEW

File: pallets/msa/src/tests.rs [hunk 1]
```diff
@@ -490,43 +490,43 @@ pub fn create_sponsored_account_with_delegation_with_valid_input_should_succeed(
...
+			add_provider_payload.clone()
 		));
 
 		// assert
 		let key_info = Msa::get_key_info(AccountId32::new(delegator_account.0));
 		assert_eq!(key_info.unwrap().msa_id, 2);
 
-		let delegated = Msa::get_delegate_info_of(Delegate(1), Delegator(2));
-		assert_eq!(delegated.is_some(), true);
+		let providerd = Msa::get_provider_info_of(Provider(1), Delegator(2));
```
Comment (shannonwells) [inline pallets/msa/src/tests.rs:517]: Overeager replacement - I'd rename this to "provider_info"

File: pallets/msa/src/tests.rs [hunk 2]
```diff
@@ -490,43 +490,43 @@ pub fn create_sponsored_account_with_delegation_with_valid_input_should_succeed(
...
+		assert_eq!(providerd.is_some(), true);
 
 		let events_occured = System::events();
 		let created_event = &events_occured.as_slice()[1];
-		let delegated_event = &events_occured.as_slice()[2];
+		let providerd_event = &events_occured.as_slice()[2];
```
Comment (shannonwells) [inline pallets/msa/src/tests.rs:522]: and this can probably be "provider_event"

**Review by shannonwells — COMMENTED** [review]
Need to change just those two things in tests is all.

File: pallets/msa/src/tests.rs (hunk 1 above)
Comment (saraswatpuneet) [reply pallets/msa/src/tests.rs:517]: :D good catch hence was this spin off , will take a keen look

**Comment by saraswatpuneet** [comment]
Merging it back to other PR, nothing else found which was out of the place

**Pull request merged**